src/css/feedback-form.css

```diff
@@ -60,14 +60,13 @@
   line-height: 40px;
 }
 
-button[type=submit]:not(.aa-SubmitButton),
-button.close {
+.submitButton {
```

Comment:
Why did we remove the original selector? It was avoiding styles for the Algolia submit button.

Reply:
Hmm I removed it because being a specific rule it was overring any other config that I had. But if it affects other buttons we can roll that back and find another solution.

Reply:
reverted